style.css

```diff
@@ -1,3 +1,224 @@
-h1 {
-  background-color: blue;
+body {
+  margin: 0;
+  padding: 0;
+  background-color: white;
+}
+
+.nav-bar {
+  position: fixed;
+  height: 8vh;
+  width: 100%;
+  background: linear-gradient(to right, #81bae6, yellow);
+}
+
+.grid-container {
+  display: grid;
+  width: 100%;
+  margin-top: 1%;
+  grid-template-columns: repeat(3, 1fr);
+  justify-content: center;
+}
+
+.item1 {
+  padding-left: 2%;
+}
+
+.grid-item2 {
+  text-align: center;
+}
+
+.txt {
+  width: 80%;
+  height: 80%;
+}
+
+.btn {
+  height: 80%;
+}
+
+.grid-item3 {
+  text-align: right;
+}
+
+.icon1 {
+  margin-right: 7%;
+}
+
+.icon2 {
+  margin-right: 7%;
+}
+
+.icon3 {
+  margin-right: 7%;
+}
```

Comment:
Please, select all of these classes at the same time and apply the property only once, since you are repeating the same.